### PR TITLE
Release Notes for 0.4.1 release.

### LINKED ISSUE #838: [Getting Started] conduit stat deployments --namespace=emojivoto
The (otherwise excellent) "Getting Started" has (I think) a bug:

https://conduit.io/getting-started/#using-the-cli

`conduit stat deployments` (no longer) displays the `emojivoto` Deployments because these are under the `emojivoto' namespace.
```
conduit stat deployments
No traffic found.
```
Recommend:
Changing the instruction to `conduit stat deployments --namespace=emojivoto` which WAI

```
conduit stat deployments --namespace=emojivoto
NAME       MESHED   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99
emoji         1/1   100.00%   2.0rps           1ms           1ms           1ms
vote-bot      1/1         -        -             -             -             -
voting        1/1    88.33%   1.0rps           1ms           1ms           2ms
web           1/1    94.02%   1.9rps           2ms           4ms           5ms
```

### PR DESCRIPTION
**Pull request by siggy**
Also update Getting Started and Debugging docs to reflect changes in
`Tap` and `Stat`.

Fixes #838

Signed-off-by: Andrew Seigner <siggy@buoyant.io>

### CODE REVIEW

**Comment by briansmith** [comment]
I think we should include a reference to "Improved discovery of `ExternalName` services" after #764 was merged.

File: doc/debugging-an-app.md [hunk 1]
```diff
@@ -59,7 +60,7 @@ requests.
 Let's figure out where those are coming from. Let's run the `tap` command again,
 and grep the output for `Unknown`s:
 
-####  ```conduit tap deploy emojivoto/voting | grep Unknown -B 2```
+####  ```conduit -n emojivoto tap deploy/voting | grep Unknown -B 2```
 
 ```
 req id=0:212 src=172.17.0.8:58326 dst=172.17.0.10:8080 :method=POST :authority=voting-svc.emojivoto:8080 :path=/emojivoto.v1.VotingService/VotePoop
```
Comment (olix0r) [inline doc/debugging-an-app.md]: would be nice, but not essential, to update the `dst` labels to be pod names in these examples

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
+## v0.4.1
+
+Conduit 0.4.1 builds on the telemetry work from 0.4.0, providing rich,
+Kubernetes-aware observability and debugging.
+
+* Web UI
+  * **New** Automatically-configured Grafana dashboards for Services, Pods,
+    ReplicationControllers, and Conduit Mesh Health. These dashboards also expose
```
Comment (wmorgan) [inline CHANGES.md]: Please lowercase "mesh health"

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+    new TCP-level stats.
```
Comment (wmorgan) [inline CHANGES.md]: Break TCP out as a separate line item. This is big!

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,32 @@
...
+  * **New** `conduit dashboard` Pod and ReplicationController views.
+* Command-line interface
+  * `conduit tap` has been completely rewritten to accept kubectl-style resources.
+    This command now supports tapping a single Kubernetes resource or group of
+    resources. The `--to` filter can be used to filter tapped requests by destination.
+  * `conduit stat` and `conduit tap` now both support `namespaces`, `deployments`,
+    `replicationcontrollers`, `services`, and `pods`, via kubectl-style resource
```
Comment (wmorgan) [inline CHANGES.md]: Does the kubectl-style resource strings make more sense in the bullet item?

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,32 @@
...
+    strings like `deploy`, `deploy/web`, and `deploy web`.
+* Telemetry
+  * **New** Tap support for filtering by and exporting destination metadata.
```
Comment (wmorgan) [inline CHANGES.md]: Is this section redundant given the CLI and web UI sections above?

Comment (siggy) [reply CHANGES.md]: it is a bit redundant, though it's true that we have new telemetry APIs, and also web and cli features that expose them. /cc @olix0r

File: CHANGES.md (hunk 3 above)
Comment (siggy) [reply CHANGES.md]: i think this depends on whether we keep the `Telemetry` section below.

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,35 @@
...
+
+* Web UI
+  * **New** Automatically-configured Grafana dashboards for Services, Pods,
+    ReplicationControllers, and Conduit mesh health.
+  * **New** `conduit dashboard` Pod and ReplicationController views.
+* Command-line interface
+  * `conduit tap` has been completely rewritten to accept kubectl-style resources.
+    This command now supports tapping a single Kubernetes resource or group of
+    resources.
+  * `conduit stat` and `conduit tap` now both support kubectl-sytle resource
```
Comment (wmorgan) [inline CHANGES.md]: type: sytle

**Pull request merged**